editoast/src/views/projects.rs

```diff
@@ -127,14 +127,13 @@ async fn create(
     data: Json<ProjectCreateForm>,
 ) -> Result<Json<ProjectWithStudies>> {
     let project: Project = data.into_inner().into();
-    if let Some(Some(image)) = project.image {
+    if let Some(image) = project.image {
         check_image_content(db_pool.clone(), image).await?;
     }
-    let project = project.create(db_pool).await?;
-    let project_with_studies = ProjectWithStudies {
-        project,
-        studies_count: 0,
-    };
+    let conn = &mut db_pool.get().await?;
+    use crate::modelsv2::Create;
```

Comment:
This should be moved in the file headers.

Reply:
Done.